ext/SymbolicsD3TreesExt.jl

```diff
@@ -0,0 +1,34 @@
+module SymbolicsD3TreesExt
+
+using Symbolics
+using SymbolicUtils
+using D3Trees
+using D3Trees.AbstractTrees
+
+function sym_nodelabel(x)
+    if x isa SymbolicUtils.BasicSymbolic
+        if SymbolicUtils.iscall(x)
+            string(SymbolicUtils.operation(x))
+        else
+            string(x)
+        end
+    else
+        string(x)
+    end
+end
+
+function D3Trees.D3Tree(t::Symbolics.Num; kwargs...)
+    symbolic = Symbolics.unwrap(t)
+    nodes = collect(AbstractTrees.PreOrderDFS(symbolic))
+    merged_kwargs = merge(
+        (
+            text = sym_nodelabel.(nodes),
+            tooltip = string.(nodes),
+            init_expand = 3
+        ),
+        kwargs
+    )
+    D3Tree(symbolic; merged_kwargs...)
+end
```

Comment:
```suggestion
function D3Trees.D3Tree(t::Symbolics.Num; kwargs...)
    symbolic = Symbolics.unwrap(t)

    merged_kwargs = merge(
        (
            detect_repeat = false,
            text = sym_nodelabel, tooltip = string,
            lazy_expand_after_depth = 8, lazy_subtree_depth = 4, init_expand = 8
        ),
        kwargs
    )

    D3Tree(symbolic; merged_kwargs...)
end
```

Reply:
Set `detect_repeat = false` , we probably don't want `D3Tree` to detect equal nodes by default (or do we? Not sure about the performance implications).

Also use new capability of D3Trees v0.3.5 to pass functions for `text` and `tooltip`. This ensures that node labels and tooltips are correct for either setting of `detect_repeat` and for lazy tree loading.